client/tests/integration/asset.rs

```diff
@@ -11,6 +11,11 @@ use iroha_primitives::fixed::Fixed;
 use serde_json::json;
 use test_network::*;
 
+#[deprecated(
+    since = "2.0.0-pre-rc.20",
+    note = "This test is deprecated, use test_register_asset_definitions.py instead"
+)]
+#[ignore = "migrated to client cli python tests"]
```

Comment:
why are these tests ignored in this PR? We should keep them around, at least for now

Reply:
> why are these tests ignored in this PR? We should keep them around, at least for now

1. We are testing the same thing and at the same integration level and it looks like redundant testing.
2. The task was set with this motivation:
> client/tests/integration were meant as integration tests, but bypass the iroha_client_cli and as such do not provide adequate coverage. Instead of having two potentially divergent branches of tests, it makes sense to consolidate.

Reply:
Although on the other hand, it may be more correct to bring сlient_сli tests to the UI level and leave the old ones.

Reply:
please leave the old tests in atm